frontend/src/lifetime/errors.ts

```diff
@@ -71,12 +71,11 @@ function githubUrl(panicDetails: string): string {
 		${browserVersion()}, ${operatingSystem()}
 
 		**Stack Trace**
-		Copied from the crash dialog in the Graphite Editor:
+		Copied from the crash dialog in the Graphite Editor:`;
 
-		\`\`\`
-		${panicDetails}
-		\`\`\`
-		`;
+	body += "\n\n```\n";
+	body += panicDetails;
+	body += "```";
```

Comment:
Missing a `\n` between `panicDetails` and the three backticks.

Reply:
panicDetails has a trailing `\n`

Reply:
![](https://files.keavon.com/-/DevotedSunnyAndeancat/capture.png)

Reply:
Maybe we should `.trim()` `panicDetails` then and manually add the `\n` if there's ever inconsistency. Maybe it's even browser-dependent. This is on Chrome.

Reply:
It does appear to be browser dependant.